src/ngraph/serializer.cpp

```diff
@@ -1151,7 +1152,14 @@ shared_ptr<Node> JSONDeserializer::deserialize_node(json node_js)
         }
         case OP_TYPEID::DynReshape:
         {
-            node = make_shared<op::DynReshape>(args[0], args[1]);
+            if (op_version == 0)
+            {
+                node = make_shared<op::v0::DynReshape>(args[0], args[1]);
```

Comment:
Looks like deserializer and was missing the zero_flag. :(

Please update the serializer as well.

Reply:
Serializer was updated.